modules/gmc-agent/src/main/java/org/gridgain/action/controller/BaselineActionsController.java

```diff
@@ -65,14 +66,61 @@ public void updateAutoAdjustAwaitingTime(long awaitingTime) {
 
     /**
      * @param ids Node ids.
-     * @return Completeble feature.
      */
     public void setBaselineTopology(Collection<String> ids) {
         authorizeIfNeeded(ctx.security(), SecurityPermission.ADMIN_OPS);
 
-        Set<UUID> uuids = ids.stream().map(UUID::fromString).collect(Collectors.toSet());
+        ctx.grid().cluster().setBaselineTopology(findNodesByConsistentIds(ids));
+    }
+
+    /**
+     * @param ids Ids.
+     */
+    private Collection<BaselineNode> findNodesByConsistentIds(Collection<String> ids) {
```

Comment:
lets rename to `buildBasline(consIds)` ?

Reply:
Renamed.